.github/workflows/tests.yml

```diff
@@ -65,8 +67,14 @@ jobs:
         id: install-tardis
         # shell: bash -l {0}
         run: |
-          pip install git+https://github.com/tardis-sn/tardis.git
-          
+            if [ "${{ github.event.action }}" == "Tardis-Dispatch" ]; then
```

Comment:
just do this using if conditional syntax of github actions right?

Reply:
We can do that as well, i thought creating two different steps might be confusing. But i will refactor this with if syntax of github actions.